hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/execution/traceability/HederaOperationTracer.java

```diff
@@ -33,4 +35,19 @@ public interface HederaOperationTracer extends HederaEvmOperationTracer {
      *     SYSTEM}
      */
     void tracePrecompileResult(final MessageFrame frame, final ContractActionType type);
+
+    /**
+     * Set the instrumentation collector for operations involving the sidecar records.
+     *
+     * @param instrumentation An instrumentation collector for EVM operations on sidecar records.
+     */
+    void setInstrumentation(@NonNull final SidecarInstrumentation instrumentation);
```

Comment:
Does it really make sense to have a setter? Looks like the instance could be passed directly to the constructor

Reply:
I must have had a reason (I _always_ have a reason) but darned if I can think of it now.  Will fix.